src/test/java/lotto/domain/LottosTest.java

```diff
@@ -1,18 +1,25 @@
 package lotto.domain;
 
+import java.util.List;
+
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@DisplayName("도메인 Lottos 테스트")
 public class LottosTest {
 
-    @DisplayName("로또 번호 6자리 생성")
+    @DisplayName("당첨 통계 산출")
     @Test
-    void lottoGenerate() {
-        Lotto result = new Lotto();
+    void computeStatistic() {
+        Lotto givenLotto = Lotto.ofNumbers(List.of(1,2,3,4,5,6));
+        Lottos lottos = new Lottos(List.of(givenLotto));
+        WinLotto winLotto = WinLotto.ofNumbers(List.of(1, 2, 3, 4, 5, 7), 6);
```

Comment:
```suggestion
Lottos lottos = 로또생성됨(List.of(1,2,3,4,5,6));
WinLotto winLotto = 당첨로또생성됨(List.of(1,2,3,4,5,6), 6);
```

생성자를 다양하게 하셔도 됩니다~
테스트픽스쳐를 활용하셔도 되구요~

[테스트 픽스처 올바르게 사용하기](https://jojoldu.tistory.com/611)

Reply:
@BeforeEche로 빼는거나 @methodsource를 사용하는것도 좋을거같네요. 테스트케이스가 하나라서 우선 생성자를 하나 추가하는걸로 라인 한 줄 줄였습니다.